config/initializers/devise.rb

```diff
@@ -236,7 +236,7 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  config.omniauth :facebook, "1495604120735583", "4adef8998ba66e2643a8258f2c74a312", scope: 'email', info_fields: 'email, name, image'
+  config.omniauth :facebook, "1495604120735583", "4adef8998ba66e2643a8258f2c74a312", scope: 'email', info_fields: 'email, name'
```

Comment:
Pushing keys to servers can be a pain in the ass. You have a few options from simple, but tedious to more complex but automated

(1) Manually update your application server code with each deploy, and relaunch the application server process
(2) Automate the above process with a script that you need only provide a list of IP addresses for
(3) Customize the cloudformation templates inject the keys into the file prior to launching the server

The third option is the most robust, but you'd have to alter each of the different cloudformation templates. Fortunately it should be the same change to each.